README.md

```diff
@@ -1,4 +1,76 @@
+
+## How to use this plugin
+
+### Installing the plugin to TeamCity
+
+1. Build the plugin using the "Building" instructions below.
+2. In your TeamCity instance go to Administration -> Diagnostics -> Internal Properties.
+    1. Ensure you update add a property `octopus.teamcity.opentelemetry.plugin.endpoint=<your_opentelemetry_collector_endpoint>`
+    2. Ensure you update add a property `octopus.teamcity.opentelemetry.plugin.headers=<your_opentelemetry_collector_endpoint_required_headers>`
+3. Alternatively open the `internal.properties` file location in your TeamCity instance `data_dir/config` folder and update the above properties
+4. Copy to .zip to your TeamCity data_dir/plugins directory and restart TeamCity server OR install the .zip using your TeamCity instance UI
+
+### Local Development
+
+#### Prerequisites
+
+##### Running Locally
+
+You must have a TeamCity instance running. To run a TeamCity instance locally:
+1. Download the TeamCity distro for your OS from https://www.jetbrains.com/teamcity/download/#section=on-premises.
+2. Unzip the distro and place in a directory of your choosing. 
+3. To run the default server, open terminal then cd in your TeamCity directory then run `./bin/teamcity-server.sh run`. An alias these commands for repeated future can be made.
+4. In a separate terminal, to run the default agent, cd in your TeamCity directory then run `./buildAgent/bin/agent.sh run`.
+5. Open localhost:8111 in a browser.
+6. On your first run create an admin login (this setup only needs to take place once due to the configuration mount). Once the server starts up, navigate to Agents->Unauthorized and authorise the agent that was started in a container alongside the server.
+7. (Optional) If attaching a remote debugger run in your TeamCity directory `export TEAMCITY_SERVER_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8111 && $SCRIPT_PATH/bin/teamcity-server.sh run` for the server and `export TEAMCITY_SERVER_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8111 && $SCRIPT_PATH/buildAgent/bin/agent.sh run` for the default agent.
+8. In your TeamCity instance go to Administration -> Diagnostics -> Internal Properties.
+   1. Ensure you update add a property `octopus.teamcity.opentelemetry.plugin.endpoint=<your_opentelemetry_collector_endpoint>`
+   2. Ensure you update add a property `octopus.teamcity.opentelemetry.plugin.headers=<your_opentelemetry_collector_endpoint_required_headers>`
+9. To stop the TeamCity server and agent from running, in a separate terminal cd to your TeamCity directory and run `./bin/runAll.sh stop`
+
+##### Using Docker
```

Comment:
Recommend moving this up to promote a "docker first" approach

Reply:
moved up!